pkg/docker/generate.go

```diff
@@ -235,7 +235,7 @@ import os
 os.chdir("` + g.getWorkdir() + `")
 sys.path.append("` + g.getWorkdir() + `")
 from ` + module + ` import ` + class + `
-cog.RedisQueueWorker(` + class + `(), redis_host=sys.argv[1], redis_port=sys.argv[2], input_queue=sys.argv[3], upload_url=sys.argv[4], consumer_id=sys.argv[5]).start()`
+cog.RedisQueueWorker(` + class + `(), redis_host=sys.argv[1], redis_port=sys.argv[2], input_queue=sys.argv[3], upload_url=sys.argv[4], consumer_id=sys.argv[5], model_id=sys.argv[6], log_queue=sys.argv[7]).start()`
```

Comment:
I wonder how we can make this "API" backward / future compatible. I expect we'll keep on adding new options, and old models should work with the new runner (and new models with old runners?).